src/main/java/org/hydev/mcpm/client/database/export/Export.java

```diff
@@ -0,0 +1,43 @@
+package org.hydev.mcpm.client.database.export;
+
+import org.hydev.mcpm.client.database.boundary.ExportPluginsBoundary;
+import org.hydev.mcpm.client.database.fetcher.DatabaseFetcher;
+import org.hydev.mcpm.client.database.fetcher.ProgressBarFetcherListener;
+import org.hydev.mcpm.client.database.inputs.ExportPluginsInput;
+import org.hydev.mcpm.client.database.results.ExportPluginsResult;
+
+import java.io.PrintStream;
+
+/**
+ * An implementation of ExportPluginsBoundary that fetches from a database.
+ */
+public class Export implements ExportPluginsBoundary {
```

Comment:
1 Use Case = 1 Interactor

```suggestion
public class ExportInteractor implements ExportPluginsBoundary {
```